rqt_plot/src/rqt_plot/rosplot.py

```diff
@@ -154,6 +154,10 @@ def _get_data(self, msg):
         val = msg
         try:
             if not self.field_evals:
+                if type(val)==std_msgs.msg.Bool:
+                    val = val.data
+                if type(val)==bool:
+                    return 1.0 if val else 0.0
```

Comment:
These two lines are not necessary. The following line already achieve the same conversion: `float(True)` -> `1.0` and `float(False)` -> `1.0`.